api/v1alpha1/apidefinition_types.go

```diff
@@ -41,6 +41,10 @@ type RoutingTriggerOnType string
 // +kubebuilder:validation:Enum=GET;POST;PUT;PATCH;DELETE;OPTIONS;HEAD;CONNECT;TRACE
 type HttpMethod string
 
+// JWTSigningMethod algorithm used to sign jwt token
+// +kubebuilder:validation:Enum=rsa;hmac;ecdsa
+type JWTSigningMethod string
+
```

Comment:
Why not leave this inside the API Definition object? Is it reused anywhere?

Reply:
> Is it reused anywhere?
Nope, removed at 4242b4f